scripts/start-happypath.sh

```diff
@@ -141,7 +140,7 @@ run_workload() {
     }
 
     log_note "run TDVM"
-    cp "$acon_root/aconcli/acon-startvm" . && ./aconcli run -n "$docker_id.json" -c :5532 || {
+    cp "$acon_root/scripts/acon-startvm" . && ATD_BIOS=OVMF.fd ATD_KERNEL=kernel.img ./aconcli run -n "$docker_id.json" -c :5532 || {
```

Comment:
Why not use -f flag to specify the acon-startvmpath? Alternatively,aconclican run directly within$acon_root/aconcli/` directory (without copying).